client/src/components/QACardAnswers.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import Modal from 'react-modal';
+import axios from 'axios';
+import QuestionsAndAnswers from './QuestionsAndAnswers.jsx';
+
+// Modal.setAppElement('#app');
+if (process.env.NODE_ENV !== 'test') {
+  Modal.setAppElement('#app');
+}
+
+const QACardAnswers = ({ currentAnswers }) => {
+
+  const [answers, setAnswers] = useState([]);
+  // toggle is to make sure we avoid infinite rerender loop
+  const [toggle, setToggle] = useState(false);
+
+  useEffect(() => {
+    if (answers) {
+      for (var key in currentAnswers) {
+        answers.push(currentAnswers[key]);
+      }
+    }
+    setToggle(true);
+  }, []);
+
+  return (
+    < div id="qa-div-card">
+      {answers.map((answer, index) =>
+        <div className="external" key={index}>
+          <div className="qa-div" ><b>A:</b> {answer.body}</div>
+            <div className="qa-footer">
+              <label className="qa-username">by {answer.answerer_name},
+                <label className="qa-date"> {answer.date}
+                  <label className="qa-helpful"> <a href="">Helpful? ({answer.helpfulness})</a>
+                    <label className="qa-report"> <a href="">Report </a>
```

Comment:
This pyramid of nested labels is a pattern I haven't seen before. Why labels inside labels, rather than:

...
<label className="qa-username">by {answer.answerer_name},</label>
<label className="qa-date"> {answer.date}</label>
...etc?

Reply:
I will refactor it to remove the labels and do it in a more cohesive way, initially set the labels up for easy readability but I think it could be condensed. I'll let you know once updated later today!